packages/puppeteer-core/src/common/bidi/BrowsingContext.ts

```diff
@@ -116,6 +117,26 @@ export class BrowsingContext extends EventEmitter {
     }
   }
 
+  async reload(options: WaitForOptions = {}): Promise<void> {
```

Comment:
does it only work on frames?

Reply:
For what else are we expecting it to work. In BiDi spec it's any browsing context.